go.mod

```diff
@@ -6,4 +6,5 @@ require (
 	github.com/blang/semver v3.5.0+incompatible
 	github.com/pkg/errors v0.8.0
 	github.com/stretchr/testify v1.3.0
+	golang.org/x/sys v0.0.0-20190812073006-9eafafc0a87e
```

Comment:
can we use the same version as k/k?
https://github.com/kubernetes/kubernetes/blob/master/go.mod#L127

`v0.0.0-20191022100944-742c48ecaeb7`

Reply:
I'll update it when cgroupv2 is actually merged.

Reply:
@KentaTada cgroup v2 support for the Kubelet was merged, I think this PR can move forward

Reply:
@giuseppe Thank you for your info. I did not know when was suitable for merge because  https://github.com/kubernetes/kubernetes/pull/89567 was still ongoing.